app/views/media/_medium.html.erb

```diff
@@ -90,7 +90,7 @@
               </h5>
             </div>
             <div>
-              <span class="badge badge-secondary"
+              <span class="<%= medium.subheader_style %>"
```

Comment:
I would suggest adding  a tooltip that explains the situation (file update etc.) in case that there is a badge-danger. Otherwise users will not know what this means.